app/controllers/api/cocktails_controller.rb

```diff
@@ -5,8 +5,8 @@ def index
         "https://www.thecocktaildb.com/api/json/v2/9973533/filter.php",
         {
           params: {
-            i: params[:q]
-          }
+            i: params[:q],
+          },
```

Comment:
Why do these commas keep coming back? ^^
Are they needed? Third time commenting about this now.

Reply:
it is my ruby formatter :))